openstack/glance/templates/glance-nanny-deployment.yaml

```diff
@@ -73,15 +69,13 @@ spec:
               name: sentry
               key: glance.DSN.python
         {{- end }}
-        {{- if .Values.api.resources.enabled }}
-        resources:
-          limits:
-            cpu: {{ .Values.api.resources.limits.cpu | quote }}
-            memory: {{ .Values.api.resources.limits.memory | quote }}
-          requests:
-            cpu: {{ .Values.api.resources.requests.cpu | quote }}
-            memory: {{ .Values.api.resources.requests.memory | quote }}
-        {{- end }}
+          resources:
+            requests:
+              memory: "250Mi"
+              cpu: "25m"
+            limits:
+              memory: "250Mi"
+              cpu: "100m"
```

Comment:
The indentation looks off. Does this produce valid YAML?

Reply:
fixed